libs/ts-rest/react-query/package.json

```diff
@@ -20,30 +20,16 @@
     "url": "https://github.com/ts-rest/ts-rest.git",
     "directory": "libs/ts-rest/react-query"
   },
+  "peerDependencies": {
+    "react": "^16.8.0 || ^17.0.0 || ^18.0.0",
+    "zod": "^3.0.0",
+    "@tanstack/react-query": "^4.0.0"
+  },
   "peerDependenciesMeta": {
     "zod": {
       "optional": true
-    },
-    "@swc/helpers": {
-      "optional": true
     }
   },
-  "peerDependencies": {
-    "react": "16.x.x || 17.x.x || 18.x.x",
-    "zod": "3.x.x",
-    "@tanstack/react-query": "4.x.x"
-  },
-  "exports": {
-    ".": {
-      "import": "./index.js",
-      "require": "./index.cjs"
-    },
-    "./package.json": "./package.json"
-  },
-  "main": "index.cjs",
-  "module": "index.js",
-  "types": "./src/index.d.ts",
-  "type": "module",
```

Comment:
These lines are automatically generated by nx after enabling `generateExportsField`. Here's how the resulting package.json looks like

```json
{
  "name": "@ts-rest/react-query",
  "version": "3.16.0",
  "description": "react-query client integration for @ts-rest",
  "license": "MIT",
  "keywords": [
    "rpc",
    "rest",
    "typescript",
    "typescript-rest",
    "react-query",
    "nestjs",
    "http",
    "api",
    "type safe"
  ],
  "private": false,
  "repository": {
    "type": "git",
    "url": "https://github.com/ts-rest/ts-rest.git",
    "directory": "libs/ts-rest/react-query"
  },
  "peerDependencies": {
    "react": "^16.8.0 || ^17.0.0 || ^18.0.0",
    "zod": "^3.0.0",
    "@tanstack/react-query": "^4.0.0",
    "@ts-rest/core": "3.16.0"
  },
  "peerDependenciesMeta": {
    "zod": {
      "optional": true
    }
  },
  "typedoc": {
    "entryPoint": "./src/index.ts",
    "tsconfig": "./tsconfig.lib.json"
  },
  "module": "./index.js",
  "main": "./index.cjs",
  "type": "module",
  "types": "./src/index.d.ts",
  "exports": {
    ".": {
      "types": "./src/index.d.ts",
      "import": "./index.js",
      "require": "./index.cjs"
    }
  },
  "dependencies": {}
}
```

Reply:
@oliverbutler I also double-checked to make sure this wouldn't cause the issue that has occurred before where it was breaking ESM imports in next.js